packages/modeling/src/operations/extrusions/project.js

```diff
@@ -48,7 +49,12 @@ const projectGeom3 = (options, geometry) => {
   projpolys = projpolys.sort((a, b) => poly3.measureArea(b) - poly3.measureArea(a))
 
   // convert polygons to geometry, and union all pieces into a single geometry
-  const projgeoms = projpolys.map((p) => geom2.fromPoints(p.vertices))
+  const projgeoms = projpolys.map((p) => {
+    // This clones the points from vec3 to vec2
+    const cloned = p.vertices.map(vec2.clone)
```

Comment:
Why doesn't geom2.fromPoints() clone the vectors?

Reply:
this should be geom2.create(cloned)

Reply:
Martinez doesn't crash, but some of the points returned will be improperly Vec3 instead of Vec2

Reply:
@platypii i suspect that this will have to be addressed later. but for now, let's go with this.